include/mbgl/gl/gl_values.hpp

```diff
@@ -76,7 +76,7 @@ struct DepthMask {
     }
 };
 
-struct ColorMask {
+struct ColorMaskValue {
```

Comment:
Renames here to avoid conflicts with new type names. I envision that all the structs in gl_values.hpp will eventually go away.